plugin/package.json

```
@@ -11,7 +11,7 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "scripts": {
-    "prepare": "yarn generate",
+    "build": "yarn generate",
     "generate": "yarn tsc && yarn bundle && yarn format && yarn clean",
```

Can we simply rename `generate` to `build`?
```suggestion
    "build": "yarn tsc && yarn bundle && yarn format && yarn clean",
```